make-plus-booster-test.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+node_version=$(cat versions.mk|grep NODE_VERSION|cut -d'=' -f2)
+npm_version=$(cat versions.mk|grep NPM_VERSION|cut -d'=' -f2)
+image_name=rhoar/local-centos-nodejs-s2i
+booster_image_name=nodejs-rest-http-s2i-rhoar
+
+make
+
+# Build the docker image
+docker build -t $image_name --build-arg NODE_VERSION=$node_version --build-arg NPM_VERSION=$npm_version .
+
+pushd ../nodejs-rest-http/
+
+s2i build . ${image_name}:latest $booster_image_name
```

Comment:
works fine even without `--pull-policy never` ?

Reply:
other than that is good to have a script like this :+1:

Reply:
Tested locally after delete all the images and it is working fine.

Reply:
Thanks @helio-frota, I'll add the `--pull-policy` to the script.